homeassistant/components/dynalite/__init__.py

```diff
@@ -198,6 +205,41 @@ async def async_setup(hass: HomeAssistant, config: Dict[str, Any]) -> bool:
             )
         )
 
+    def get_bridges(host: str) -> List[DynaliteBridge]:
+        result = []
+        for entry_id in hass.data[DOMAIN]:
+            cur_bridge = hass.data[DOMAIN][entry_id]
+            if not host or cur_bridge.host == host:
+                result.append(cur_bridge)
+        return result
+
+    async def request_area_preset_service(service_call: ServiceCall):
+        LOGGER.debug(
+            "Received service call service=%s data=%s",
+            service_call.service,
+            service_call.data,
+        )
+        host = service_call.data.get(CONF_HOST, "")
+        bridges = get_bridges(host)
+        LOGGER.debug("Selected bridged for service call: %s", bridges)
+        area = service_call.data[CONF_AREA]
+        channel = service_call.data.get(CONF_CHANNEL, None)
```

Comment:
`None` is the default value returned if the key is missing in the dict.

Reply:
ok. changed.